### DISCUSSION TITLE
Conceal with treesitter

### DISCUSSION THREAD

**Discussion by sarmong** [nvim-treesitter/nvim-treesitter#2226] [General]
Is it possible to do `conceal` with treesitter?

This is usually done using `:syntax`, but it accepts regex. Can I do smth like this by providing treesitter node?

![image](https://user-images.githubusercontent.com/42828375/148687154-99b703eb-fc9e-400a-a076-cde601044019.png)
![image](https://user-images.githubusercontent.com/42828375/148687171-4b1985fe-19a4-467d-8939-05ee09984f9b.png)

**Comment by theHamsta** [comment]
No, tree-sitter in Neovim deactivates all functionalities of `:h syntax` and there's no replacement in tree-sitter for syntax concealment yet.

**Reply by vigoux** [reply]
But this is in the plans of things to add, so keep yourself up to date !

**Reply by sarmong** [reply]
Thanks, is there an issue I can follow?
